doc/man3/SSL_CTX_set_verify.pod

```diff
@@ -43,6 +48,15 @@ verification that shall be allowed for B<ctx>.
 SSL_set_verify_depth() sets the maximum B<depth> for the certificate chain
 verification that shall be allowed for B<ssl>.
 
+SSL_force_post_handshake_auth() forces the Post-Handshake Authentication extension
+to be added to the ClientHello regardless of certificate configuration at the time
+of the initial handshake, such that post-handshake authentication can be requested
+by the server.
+
+SSL_verify_client_post_handshake() causes a Certificate Request message to be sent
+by a server on the given B<ssl> connection. The SSL_VERIFY_PEER flag must be set,
+the SSL_VERIFY_POST_HANDSHAKE flag is optional.
```

Comment:
The `SSL_VERIFY_POST_HANDSHAKE` flag blocks certificate verification in the initial handshake. It can be used to configure an SSL_CTX or SSL before establishing the connection, rather than setting the flags later when calling `SSL_verify_client_post_handshake()`. It's also a convenience for unit tests; removing it would complicate the unit tests.

Reply:
Ah, ok. I understand it now. The docs do say that, but somehow I didn't get that understanding from them. Probably that's just me.

Reply:
Should the docs say what happens if you try this on a non TLSv1.3 connection?